samples/client/petstore/swift3/default/PetstoreClient/Classes/Swaggers/APIHelper.swift

```diff
@@ -8,27 +8,25 @@ import Foundation
 
 class APIHelper {
     static func rejectNil(_ source: [String:Any?]) -> [String:Any]? {
-        var destination = [String:Any]()
-        for (key, nillableValue) in source {
-            if let value: Any = nillableValue {
-                destination[key] = value
-            }
-        }
-
-        if destination.isEmpty {
-            return nil
+        let result = source.reduce([String:Any]()) { (dict, tuple:(key: String, value: Any?)) -> [String:Any] in
+            guard let value = tuple.value else { return dict }
+            var dict = dict
+            dict.updateValue(value, forKey: tuple.key)
+            return dict
         }
-        return destination
+        
+        if result.isEmpty { return nil }
+        return result
```

Comment:
I'm afraid I don't, it was as result of a few tests I run on my machine. 

But actually I doubt this method is really useful.
In Swift3 (on Swift2 as well, but I'm not sure, I don't remember) if you have:
```
var dict:[String:Any?] = [:]
dict["1"] = nil
dict["2"] = nil
print(dict)
print(dict["1"])
print(dict["3"])
print(dict.isEmpty)
print(dict.count
```
the output would be:
```
[:]
nil
nil
true
0
```

I think this method could be only: ` return source.isEmpty ? nil : source`

What do you think?